hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/FileSizeCountTask.java

```diff
@@ -96,7 +96,9 @@ private static int nextClosestPowerIndexOfTwo(long dataSize) {
       return new ImmutablePair<>(getTaskName(), false);
     }
     // Truncate table before inserting new rows
-    dslContext.truncate(FILE_COUNT_BY_SIZE);
+    int execute = dslContext.delete(FILE_COUNT_BY_SIZE).execute();
```

Comment:
Why does truncate not work?

Reply:
From Jooq doc.

`  * Most dialects implement the <code>TRUNCATE</code> statement. If it is not
     * supported, it is emulated using an equivalent <code>DELETE</code>
     * statement. This is particularly true for these dialects:
     * <ul>
     * <li> {@link SQLDialect#FIREBIRD}</li>
     * <li> {@link SQLDialect#INGRES}</li>
     * <li> {@link SQLDialect#SQLITE}</li>
     * </ul>`